README.md

```diff
@@ -54,9 +54,10 @@
 
 # LTS Plan
 
-New semver-major releases of Node.js are cut from `master` every six months.
-New even-numbered versions (e.g. v6, v8, v10, etc) are cut in April. New 
-odd-numbered versions (e.g. v5, v7, v9) are cut in October.
+New semver-major releases of Node.js are cut from `master` every six months
+and become the "current" release.  New even-numbered versions (e.g.  v6, v8,
+v10, etc) are cut in April.  New odd-numbered versions (e.g.  v5, v7, v9)
+are cut in October.
```

Comment:
>Any info on the Current branch should be in the main repo IMO.

You mean because the LTS repo should only contain info about the LTS branches? It seems weird to split up the current and LTS stuff, not least because current is sort-of defined as halfway between `master` and `lts`. Would you document what the current branch is without documenting the LTS branches?

At the moment the only documentation that current branches are a thing is in this README (as far ask I know).